src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -651,8 +652,8 @@ def post_request(
         ``data`` a dictionary of key-value pairs that will be urlencoded
                and sent as POST data
                or binary data that is sent as the raw body content
-               or passed as such for multipart form data if ``files`` is also
-                  defined
+               or passed as such for multipart form data if ``files`` is also defined
+               or file descriptor retreived by Get File Descriptor
```

Comment:
Is this true only for POST?

Reply:
Alright, according to the docs it should also be PATCH and PUT

Reply:
in theory requests lib let you pass data to all method but I think it's fine for the moment